modules/rosa-sts-arn-path-customization-for-iam-roles-and-policies.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-with-customizations.adoc
+
+:_content-type: CONCEPT
+[id="rosa-sts-arn-path-customization-for-iam-roles-and-policies_{context}"]
+= ARN path customization for IAM roles and policies
+
+When you create the AWS IAM roles and policies required for {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), you can specify custom Amazon Resource Name (ARN) paths. This enables you to use role and policy ARN paths that meet the security requirements of your organization.
+
+You can specify custom ARN paths when you create your OCM role, user role, and account-wide roles and policies. When you create the cluster-specific Operator roles, the ARN path for the relevant account-wide installer role is automatically detected and applied to the Operator roles.
```

Comment:
I believe this might confuse clients into thinking they can specify a different path for each account wide roles and policies.

"You can specify custom ARN paths when you create your OCM role, user role, and account-wide roles and policies. During the creation of the latter, a single path is chosen and the account roles and policies are setup using the path in question. When you create the cluster-specific Operator roles, the ARN path from the account installer role is automatically detected and applied to the Operator roles and it's policies"

What do you think?

Reply:
Thank you for highlighting this and for your suggestion. I will wait for the other reviews to be completed and then I will have a think about this.

Reply:
I have now updated this section to make this point very clear.